pkg/injection/codemodule/installer/symlink/symlink.go

```diff
@@ -70,9 +70,29 @@ func CreateForLatestVersion(fs afero.Fs, dk dynakube.DynaKube, latestVersionDir,
 	return nil
 }
 
+func CreateForPodInfoDir(fs afero.Fs, targetDir, symlinkDir string) error {
```

Comment:
Can't we have 1 function like `Create`?

The only difference is the log lines, and only a bit.
If we log the paths we wanted to use for the symlink creation, we can figure out which symlink it is.

Reply:
fair enough, on it

Reply:
have a look for yourself, maybe it is overkill but at least we only have one function 😅 

https://github.com/Dynatrace/dynatrace-operator/pull/4215/commits/c99035075c8164702a9f71eb830f555ec3d83b6b

Reply:
went with option b. as it seems cleaner to me.

I tested it but if you find time I would appreciate if you could test it as well

https://github.com/Dynatrace/dynatrace-operator/pull/4215/commits/8b10eaf288a9f5cd3272bb5b110df59897929fe2